recruiter-email.txt

```diff
@@ -0,0 +1,14 @@
+<Name>,
+
+Just wanted to reach out about your first CoffeeOps meet-up last Friday.
+
+Seattle CoffeeOps is a community-driven meet-up run by engineers. As such, we try to keep our meetings non-commercial and pitch-free. I'm sorry I didn't warn you of that when you gave me your staffing pitch at the beginning of the meeting.
```

Comment:
This is perhaps a bit too specific to the last occurence. Perhaps re-wording to be a bit more flexible, something like this?

_**<As a reminder|As we discussed|As a head's up>,**_ Seattle CoffeeOps is a community-driven meet-up run by engineers. As such, we try to keep our meetings non-commercial and pitch-free.